data_model/src/block.rs

```diff
@@ -79,7 +81,10 @@ mod committed {
 
     use super::*;
 
-    declare_versioned_with_scale!(VersionedCommittedBlock 1..2, Debug, Clone, FromVariant, IntoSchema, Serialize, Deserialize, PartialEq, Eq, Hash);
+    #[cfg(any(feature = "ffi_import", feature = "ffi_export"))]
```

Comment:
As I can see, we have no feature called `ffi_import`

Reply:
yes we do, it's just not finished yet so is inert